src/assets/fabricator/styles/base/variables.css

```diff
@@ -0,0 +1,11 @@
+:root {
+	--color-nav-background: #444;
+	--color-nav-group-background: #333;
+	--color-nav-text: #fff;
+	--color-item-border: #ccc;
+	--color-control-text: #0074D9;
+
+	--nav-width: 14rem;
+}
```

Comment:
What do you think about the [SUITCSS convention](https://github.com/suitcss/components-button/blob/master/lib/button.css#L3-L10) of grouping component-specific variables with namespaces? I think the `--color-` prefix makes sense for _global_ definition (e.g. `--color-red`), but when a variable is explicitly tied to a component (`nav-`, `item-`) I think it's clearer to prefix them with _that_ instead.

Example:

```css
:root {
  --Nav-background: #444;
  --Nav-color: #fff;
  --Nav-width: 14rem;
  --Item-border: 1px solid #ccc;
}
```